### ISSUE TITLE
[Bugfix 2.0.x] Printing does not start because E1 does not reach the targetted temp + Thermal runaway

### ISSUE THREAD

**Issue by Bergerac56** [MarlinFirmware/Marlin#8928]
As already mentionned in another "issue" (#8684), "I have also another issue with the 2 last bugfix 2.0: The temperature of the extruder never reaches the defined temp (If it is 200C, it "blocks" at +- 197C and never start to print) "

@Roxy-3D mentionned to have also noticed that.

It seems that the problem started somewhere here: 794f8f132966778c9c9395ff922ee5a3a5d81ef0 when K1 was adapted to PID_K1

As of today, it can ask a lot of time before E1  reaches the desired temp and the printer starts to print. (more than 10 min or never...)

**Comment by Bergerac56** [comment]
We can add thermal runaway to the problem. On  long prints, I got twice this. I suspect that, because the targeted temp is difficult to reach, it happens that  it triggers the thermal protection

**Comment by Bergerac56** [comment]
Looking at how to solve the issue, I noticed that "spikes" reappeared in the control of the mosfet of E1. (Around the introduction of commit 794f8f132966778c9c9395ff922ee5a3a5d81ef0). This induced a behavior similar to the one due to the poor converters of the RE-ARM, which was solved by the filters made by @p3p (and addition of inductors in the wiring of thermistors in my case).

In the commit list, I saw that @p3p proposed a new version of the filters. I gave a try to his commit ca4817e70adc9eeef255444994dde3ffd2050b81.

And it solves the issue. Or the filters are better or the values used by it are more accurate. The result is that there is no strange behavior in the control of the mosfet of E1 anymore. 

Hope that it will be quickly "pushed" into bugfix 2.0.  Thank you @p3p ;)
